src/script/components/panel/UserActions.tsx

```diff
@@ -310,7 +313,11 @@ const UserActions: React.FC<UserActionsProps> = ({
     removeUserFromConversation,
   ].filter((item): item is MenuItem => !!item);
 
-  return <PanelActions items={items} />;
+  return items.length === 1 && isModal ? (
+    <SingleAction item={items[0]} onCancel={onAction} />
```

Comment:
the `onAction` method passed here is the `hide` method from the `UserModal` component, it closes the modal